### ISSUE TITLE
Feature: Save NFT image to phone

### ISSUE THREAD

**Issue by seabornlee** [AlphaWallet/alpha-wallet-android#2875]
When I got a new NFT in my wallet, I want to set it as my avatar in other apps, I can only take a screenshot and crop to get the original image.

**Comment by JamesSmartCell** [comment]
Good idea @seabornlee. Should this be like a save intent for the original image?
